main/getting-started/intro-hackathon.md

```diff
@@ -0,0 +1,103 @@
+  transfer, and manipulate digital assets. 
+2. **[Zoe Introduction](./intro-zoe.md)**: 
+  This explains the concepts and components of Zoe, Agoric's reusable smart contract 
+  for writing other smart contracts in JavaScript. Zoe implements our offer safety 
+  guarantee that users either get what they wanted from the contract or a full refund
+  of what they put up for it. 
+3. **[Agoric Platform/Stack](/platform/README.md)**: A brief introduction to the complete Agoric platform/technical stack.
+4. **[Agoric Dapp Guide](/dapps/README.md)**: 
+   A Dapp is a distributed application, typically with a browser-based user interface, a public
+   API server, and a contract running on the Agoric blockchain. This document explains a Dapp's
+   basic directory and file structure.
+
+There are several ways you can go from here:
+- Watch videos showing how to write smart contracts on the Agoric platform.
+- Read the more detailed Guides and look at our APIs.
+- Look at our already written sample contracts. 
+
+For general help, you should be aware of and make use of:
+- **[Agoric Glossary](/glossary/)**: Our Glossary focuses on terms we've given Agoric-context
+  definitions to (i.e. what does *mint* mean in an Agoric context?).
+
+## Videos
+
+We have two videos of talks given to external audiences that are particularly useful to contract developers.
+
+- [DeFi Workshops: Smart Contracts in JavaScript](https://www.youtube.com/watch?v=qudVWjSqDJU)
+  by Dean Tribble covers general concepts with examples.
+  
+- [Building a Composable DeFi Contract](https://www.youtube.com/watch?v=faxrecQgEio) by Kate Sills walks through programming first a covered call option contract,
+and then writing an OTC Desk contract that uses the covered call contract.
+
+## ERTP and Zoe Guides and APIs
+
+For a deeper explanation of Agoric concepts, what you can accomplish with a contract,
+and what Agoric API commands do, read the following:
+
+- ERTP: The **[ERTP Guide](/ertp/guide/README.md)** is a detailed description of 
+  ERTP concepts, design, components, and commands. The ERTP API is documented
+  [here](/ertp/api/#ertp-api). 
+- Zoe: The **[Zoe Guide](/zoe/guide/README.md)** is a detailed description of 
+  Zoe concepts, design, components, and commands. The Zoe API is documented
+  [here](/zoe/api/#zoe-api).
+  
+## Example contracts
+
+To familiarize yourself with working Agoric smart contracts, take a look at our 
+[sample contracts](/zoe/guide/contracts/README.md). They are:
+- [Atomic Swap](/zoe/guide/contracts/atomic-swap.md)
+- [Autoswap](/zoe/guide/contracts/autoswap.md)
+- [Barter Exchange](/zoe/guide/contracts/barter-exchange.md)
+- [Covered Call](/zoe/guide/contracts/covered-call.md)
+- [Multipool Autoswap](/zoe/guide/contracts/multipoolAutoswap.md)
+- [Second-price auction](/zoe/guide/contracts/second-price-auction.md)
+- [Simple Exchange](/zoe/guide/contracts/simple-exchange.md)
+
```

Comment:
We have many more contracts than this. Can we add them to the list and link to Github if there is not yet adequate documentation?